### ISSUE TITLE
ReferenceError: file is not defined

### ISSUE THREAD

**Issue by MohGanji** [cdimascio/node-mongodb-fixtures#11]
If I don't export an array in my collection.js file, then I must be confronted with an error, instead I get this error: 
```
(.../node-mongodb-fixtures/lib/index.js:171:66) ReferenceError: file is not defined
```

I took a look at the source code and looks like you are using a variable named `file` that is not defined in the function, so maybe use the collection name instead, or pass the file to it!

here is the part of code in `lib/index.js`:

```
function bulkImportDocuments(db, mute) {
  return collectionName => docs => {
    if (!Array.isArray(docs)) {
      // prettier-ignore
      throw new Error( '[error] no docs returned from file: "' + file + // <<------- HERE
          '". verify that a docs array was exported. note: if using .js files, use "module.exports", instead of "exports".'
      );
    }
...
```

**Comment by cdimascio** [comment]
@MohGanji thanks for the ticket. i will have a look

**Comment by cdimascio** [comment]
Fixed in `v3.0.5`
